lib/SwipeableWrapper/index.jsx

```diff
@@ -61,7 +59,7 @@ const SwipeableWrapper = forwardRef(
         rAF(() => {
           onSlideChange(index.current);
           scrollToTop();
-          const height = innerHeight - elementRef.current.clientTop;
+          const height = window.innerHeight - elementRef.current.clientTop;
```

Comment:
What happens on SSR where `window` object is not defined?

Reply:
Also, use optional chaining for elementRef.

Reply:
> What happens on SSR where `window` object is not defined?

We are using `window` object only inside functions that are used inside `useLayoutEffect` which will not be called on SSR.